test/azure/terraform_azure_vm_example_test.go

```diff
@@ -0,0 +1,191 @@
+// +build azure azureslim,compute
+
+// NOTE: We use build tags to differentiate azure testing because we currently do not have azure access setup for
+// CircleCI.
+
+package test
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-07-01/compute"
+	"github.com/gruntwork-io/terratest/modules/azure"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTerraformAzureVmExample(t *testing.T) {
+	t.Parallel()
+
+	// subscriptionID is overridden by the environment variable "ARM_SUBSCRIPTION_ID"
+	subscriptionID := ""
+	uniquePostfix := random.UniqueId() // "resource" //
```

Comment:
NIT: This comment doesn't quite make sense to me. Either expand, or remove.

Reply:
Removed.. it was just there to replace the UniqueId when using Terraform manual deployment since it took longer to spin up and destroy these resources.